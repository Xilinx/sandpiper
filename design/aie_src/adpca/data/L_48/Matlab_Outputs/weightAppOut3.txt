9.665858746e-01 -3.967333794e+00
4.358505726e+00 -1.178049278e+01
-2.883377075e-01 -4.443087578e+00
