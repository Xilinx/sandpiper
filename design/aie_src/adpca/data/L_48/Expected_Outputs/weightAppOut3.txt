9.666002989e-01 -3.967265129e+00 
4.358659744e+00 -1.178008652e+01 
-2.882672548e-01 -4.442849159e+00 
